.pre-commit-config.yaml

```diff
@@ -4,6 +4,7 @@ repos:
     hooks:
       - id: "flake8"
         name: "Python: analysis"
+        stages: [manual]
```

Comment:
`all` might be more descriptive, since it runs all checks?

Reply:
See https://pre-commit.com/#confining-hooks-to-run-at-certain-stages